webapp/packages/core-authentication/src/AuthProvidersResource.ts

```diff
@@ -89,14 +89,17 @@ export class AuthProvidersResource extends CachedMapResource<string, AuthProvide
   protected async loader(originalKey: ResourceKey<string>): Promise<Map<string, AuthProvider>> {
     const all = this.aliases.isAlias(originalKey, CachedMapAllKey);
 
-    const { providers } = await this.graphQLService.sdk.getAuthProviders();
+    try {
+      const { providers } = await this.graphQLService.sdk.getAuthProviders();
+
+      const key = resourceKeyList(providers.map(provider => provider.id));
+      if (all) {
+        this.replace(key, providers);
+      } else {
+        this.set(key, providers);
+      }
+    } catch {};
 
-    const key = resourceKeyList(providers.map(provider => provider.id));
-    if (all) {
-      this.replace(key, providers);
-    } else {
-      this.set(key, providers);
-    }
     return this.data;
   }
```

Comment:
please don't wrap code in resource's loader with try catch, resources have their own error catching mechanism